server/src/modules/repo/maps-repo.service.ts

```diff
@@ -159,7 +159,36 @@ export class MapsRepoService {
     }
     //#endregion
 
+    //#region Map Stats
+    async updateMapStats(mapID: number, data: Prisma.MapStatsUpdateInput): Promise<MapStats> {
+        return this.prisma.mapStats.update({
+            where: {
+                mapID: mapID
+            },
+            data: data
+        });
+    }
+    //#endregion
+
     //#region MapTrack
+    async getMapTracks(mapID): Promise<MapTrack[]> {
+        return this.prisma.mapTrack.findMany({
+            where: {
+                mapID: mapID
+            },
+            include: {
+                zones: {
+                    include: {
+                        triggers: {
+                            include: {
+                                properties: true
+                            }
+                        }
+                    }
+                }
+            }
+        });
+    }
```

Comment:
We should be able to pass in a Prisma include here, we don't want this to hardcode the include. In some cases like creates I don't mind always returning the same include, but this seems like something likely to be used in a bunch of places and likely to want various different includes.